src/common/filters/RegexFilter.h

```diff
@@ -7,16 +7,40 @@
 #include <sstream>
 #include <regex>
 
+#include <iostream>
+
+namespace {
+
+    static const cali::ConfigSet::Entry s_configdata[] = {
+      { "regex", CALI_TYPE_STRING, "",
+        "Regular expression for matching annotions",
+        "Regular expression for matching annotions"
+      },
+      cali::ConfigSet::Terminator
+    };
+
+}
+
 class RegexFilter : public Filter<RegexFilter> {
+  private:
+    static std::string regex;
+    static cali::ConfigSet config;
+
   public:
+    static void initialize()
+    {
+      config = cali::RuntimeConfig::init("nvvp", s_configdata);
```

Comment:
Eventually the config needs to be provided by whoever instantiates a RegexFilter. We can leave it in as a hack for now.